src/components/BookstoreApp.jsx

```diff
@@ -0,0 +1,20 @@
+import { Route, Routes } from 'react-router-dom';
+import Categories from '../routes/Categories';
+import Home from '../routes/Home';
+import NotMatch from '../routes/NoMatch';
+import NavBar from './NavBar';
+
+function BookstoreApp() {
+  return (
+    <Routes>
+      <Route path="/" element={<NavBar />}>
+        <Route index element={<Home />} />
+        <Route path="books" element={<Home />} />
+        <Route path="categories" element={<Categories />} />
+        <Route path="*" element={<NotMatch />} />
+      </Route>
+    </Routes>
```

Comment:
- [ ] I think you need to call the right component. Because for now, I am not able to see any UI other than the nav bar.

![Screenshot from 2023-03-12 08-55-20](https://user-images.githubusercontent.com/26767383/224522612-af5e51e1-7360-45ac-b645-2d1163a3eb9f.png)